x/validators/errors.go

```diff
@@ -10,13 +10,17 @@ import (
 // ABCI Response Codes
 // x/update_validators reserves 40 ~ 49.
 const (
-	CodeEmptyDiff uint32 = 40
-	CodeWrongType        = 41
+	CodeEmptyDiff         uint32 = 40
```

Comment:
:nail_care:  They are `const`, so it might be better to leave them without a type. Compiler can do an extra work for us then.